### ISSUE TITLE
Password disguise in unattend.xml causes rendering errors on some hosts

### ISSUE THREAD

**Issue by tek0011** [kireevco/wimaging#33]
The setting for hiding windows passwords in the unattend seems to cause rendering errors with some hosts: 

http://i.imgur.com/IqUUZoH.png

https://github.com/kireevco/wimaging/commit/c4845d74b0f185816a6d80e0f4848a86d8c5307f#diff-062fb7b14a626f8acee9bd2053c33728R38

**Comment by helge000** [comment]
Its a long time, sorry (But seen this now in the process for integrating windows 10 and subscribing now...)

Can you pls be more specific?

**Some notes:**
1. Rendering issues are 'normal' (and ok I think) if OS != windows, since there is no regex to check if the password value is really base64 beause
2. Using the base64 password setting in Foreman is convenient, but discouraged since you need to disable safe mode rendering

**Comment by tek0011** [comment]
Its been awhile, but I believe the issues stems from Foreman itself not recognizing an empty password field in order to use the settings -> default encrypted password.  Unfortunately, I dont completely remember this issue.  I believe in Foreman, on hosts that already exist, you have to click on the little pencil button to make the password field empty.  

ie, if memory serves me, this isn't a wimaging issue, but more so how it works with Foreman.

**Comment by helge000** [comment]
Foreman does not let you create hosts without root password, AFAIK.

Most likely these root passwords where from older hosts using the default hashing (*sha256*); some nasty characters end up as password (`puts Base64.decode64(Digest::SHA256.hexdigest('test'))`)
You can verify it with this simple template:
```erb
<%= root_pass %>
```
If this is case, you can fix this by:
1. Set the correct hashing algo (Base64) in the OS
2. Edit the host and set a new root password

**Comment by tek0011** [comment]
Yep.  Without even trying, I can tell you that will work.  I attempted the same back when I was messing with all this.  I think what you say is correct.  This only happens on hosts that are already in Foreman (since the password field is empty, but not necessarily blank).   Long as you click on the pencil button of the password to edit it, and leave it blank, all is happy.

Step 2 is what we ended up going with.  

Thanks.
